<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextFilterWidget</class>
 <widget class="QWidget" name="TextFilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>38</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Text filter</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="textEdit"/>
   </item>
   <item>
    <widget class="QComboBox" name="serchTypeCombobox"/>
   </item>
   <item>
    <widget class="QCheckBox" name="caseSensitiveCheckBox">
     <property name="text">
      <string>Case sensitive</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
